botcity/web/bot.py

```diff
@@ -46,9 +46,13 @@
 
 
 def _cleanup(bot: ReferenceType[WebBot]):
+
     if bot() is not None:
         try:
+            options = bot().options
             bot().stop_browser()
+            temp_dir = options._botcity_temp_dir
+            shutil.rmtree(temp_dir, ignore_errors=True)
```

Comment:
Must check if it is `None` before deleting:
```suggestion
            temp_dir = options._botcity_temp_dir
            if not temp_dir: return None
            shutil.rmtree(temp_dir, ignore_errors=True)
```